src/main/java/org/ibp/api/java/impl/middleware/common/validator/AttributeValidator.java

```diff
@@ -27,11 +48,111 @@ public void validateAttributeIds(final BindingResult errors, final List<String>
 		final List<Attribute> attributes = this.germplasmDataManager.getAttributeByIds(ids);
 		if (attributes.size() != ids.size()) {
 			errors.reject("attribute.invalid", "");
+		}
+	}
+
+	public void validateAttributeType(final BindingResult errors, final String attributeType) {
+		if (StringUtils.isEmpty(attributeType)) {
 			return;
 		}
+
+		if (!AttributeValidator.ALLOWED_ATTRIBUTE_TYPES.contains(attributeType.toUpperCase())) {
```

Comment:
Can be consolidated to one if: 
`if (StringUtils.isEmpty(attributeType) && !AttributeValidator.ALLOWED_ATTRIBUTE_TYPES.contains(attributeType.toUpperCase())) `